### DISCUSSION TITLE
Next.js 13 + app folder (React Server Component)

### DISCUSSION THREAD

**Discussion by andrewwilkin** [RevereCRE/relay-nextjs#80] [Q&A] [answered]
Having read (and re-read) this documentation and trying to get my head around the relay with its learning curve, feel I'm making progress but then probably not 😄 

Not sure how this would work with the app folder in Next.js 13 with React Server Components. Have you had any thoughts on how this will work with future releases, or is the steer to stick with pages?

This page seems to suggest not fetching data in a client side component:
https://beta.nextjs.org/docs/rendering/server-and-client-components

**Comment by sibelius** [comment, marked as answer]
Check this https://github.com/relayjs/relay-examples/tree/main/issue-tracker-next-v13

**Reply by andrewwilkin** [reply]
Awesome, looking now 👀

**Comment by rrdelaney** [comment]
Would definitely recommend checking out the link posted by @sibelius! There's no plans on supporting anything but `getInitialProps` at the moment, although we are doing some internal refactoring to make it possible to support other patterns.
